docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -67,17 +91,43 @@ You must reload the page after disabling extensions or display issues may occur.
 
 :::
 
-## Rolling Back Extensions
+## Deleting Helm Charts
 
-Under the **Installed** tab, click the **Rollback** button on the extension you wish to roll back.
+1. Click **☰**, then click on the name of your local cluster.
+1. From the sidebar, select **Apps > Installed Apps**.
+1. Find the name of the chart you want to delete and select the checkbox next to it. 
+1. Click **Delete**.
 
-![Roll back extensions](/img/roll-back-extension.png)
+## Updating and Upgrading Extensions
 
-:::caution
+1. Click **☰ > Extensions** under **Configuration**.
+1. Select the **Updates** tab. 
+1. Click **Update**.
 
-You must reload the page after rolling back extensions or display issues may occur.
+If there is a new version of the extension, there will also be an **Update** button visible on the associated card for the extension in the **Available** tab.
 
-:::
+### Updating and Upgrading an Extensions Repository in an Air-gapped Environment
+
+Extensions repository that aren't air-gapped are automatically updated. If the repository is air-gapped, you must update it manually.
+
+First, mirror the latest changes to your private repository by following the same steps for initially [importing and installing an extension repository](#importing-and-installing-extensions-in-an-air-gapped-environment).
+
+After you mirror the latest changes, follow these steps:
+
+1. Click **☰ > Local**.
+1. From the sidebar, select **Workloads > Deployments**.
+1. From the namespaces dropdown menu, select **All Namespaces**.
```

Comment:
It is unnecessary to select All Namespaces, you should just select the `cattle-ui-plugin-system` namespace.